CMakeLists.txt

```diff
@@ -24,7 +24,7 @@
 
 cmake_minimum_required(VERSION 3.2)
 include(CheckCXXSymbolExists)
-project (sqlpp11-connector-mysql)
+project (sqlpp11-connector-mysql VERSION 0.29)
```

Comment:
The version is defined by the tag in git. Maintaining it separately in a file is not an option.

I know myself: I will forget it starting in 0.30

Reply:
Fair enough, https://github.com/rbock/sqlpp11/blob/develop/CMakeLists.txt#L28 😛 
I will check if there is something, probably some cmake module which can read the version from tag and set that up during compile time.